packages/api-client/src/granules.ts

```diff
@@ -15,21 +15,30 @@ type AssociateExecutionRequest = {
   executionArn: string
 };
 
+const encodeGranulesURIComponent = (
+  granuleId: string,
+  collectionId: string | undefined
+): string =>
+  (collectionId
+    ? `/granules/${encodeURIComponent(collectionId)}/${encodeURIComponent(granuleId)}`
+    : `/granules/${encodeURIComponent(granuleId)}`); // Fetching a granule without a collectionId is supported but deprecated
+
 /**
  * GET raw response from /granules/{granuleId} or /granules/{collectionId}/{granuleId}
  *
- * @param {Object} params                                - params
+ * @param {object} params                                - params
```

Comment:
There's conversation in Slack about docstrings generally. The consensus is that adding typings in docstrings in TypeScript files is probably redundant and prone to going stale (as seen here). I think we can generally remove type annotations from the docstrings and _maybe_ param descriptions in the case of (for example) `params.granuleId = a granule ID`.

I'd be inclined to at least remove the types here as we go but open to discussion. This idea hasn't really be codified.

Reply:
Sounds good. My goal here was to eliminate all of the linter warnings and update things where it was obviously wrong or the change was obvious. I'll definitely remove the types for now.

Reply:
Pushed a change that removes the types for now.

Reply:
Cool. We need to codify our intention with these docstrings. I think this is the right idea but we're sort of all over the place.